subprojects/ide-native/src/integTest/groovy/org/gradle/ide/visualstudio/VisualStudioSoftwareModelMultiProjectIntegrationTest.groovy

```diff
@@ -557,7 +555,7 @@ class VisualStudioSoftwareModelMultiProjectIntegrationTest extends AbstractVisua
     }
 
     /** @see IdePlugin#toGradleCommand(Project) */
-    @IgnoreIf({!(GradleContextualExecuter.daemon || GradleContextualExecuter.noDaemon)})
+    @Requires(IntegTestPreconditions.IsDaemonOrNoDaemonExecutor)
```

Comment:
For a non-insider this looks like a always true condition, i.e., run if `A || !A`